src/main/java/org/opencds/cqf/tooling/utilities/ResourceUtils.java

```diff
@@ -211,10 +211,14 @@ public static FhirContext getFhirContext(FhirVersion fhirVersion) {
     private static Map<String, IBaseResource> getR4DepLibraryResources(String path, Map<String, IBaseResource> dependencyLibraries, FhirContext fhirContext, Encoding encoding, Boolean versioned) {      
       List<String> dependencyLibraryPaths = getR4DepLibraryPaths(path, fhirContext, encoding, versioned);
       for (String dependencyLibraryPath : dependencyLibraryPaths) {
-        Object resource = IOUtils.readResource(dependencyLibraryPath, fhirContext);
-        if (resource instanceof org.hl7.fhir.r4.model.Library) {
-          org.hl7.fhir.r4.model.Library library = (org.hl7.fhir.r4.model.Library)resource;
-          dependencyLibraries.putIfAbsent(library.getId(), library);
+        if (dependencyLibraryPath.contains("ModelInfo")) {
+          System.out.println("skip");
```

Comment:
Should use logger.debug or logger.trace for this. It's generally considered bad practice to white debug output to stdout.